src/main/java/io/mosip/mimoto/controller/AttestationServiceController.java

```diff
@@ -56,6 +57,7 @@ public ResponseEntity<Object> processOffline(@RequestBody String attestation)
      * @param attestation
      * @return
      */
+    @Operation(hidden = true)
```

Comment:
why this is Operation but not ApiOperation

Reply:
@APiOperation is replaced with Operation is swagger openapi3